pkg/services/rendering/rendering.go

```diff
@@ -217,7 +217,7 @@ func (rs *RenderingService) getFilePathForNewImage() (string, error) {
 }
 
 func (rs *RenderingService) getURL(path string) string {
-	if rs.Cfg.RendererUrl != "" {
+	if rs.Cfg.RendererCallbackUrl != "" {
```

Comment:
I do not think this change looks correct. RendererCallbackUrl is always set to something so this will break the existing behavior

Reply:
Thanks, I didn't realize that. I'll have to reconsider my approach.